collections/Laud_lat/MS_Laud_Lat_103.xml

```diff
@@ -6,16 +6,17 @@
             <title>MS. Laud Lat. 103</title>
             <title type="collection">MSS. Laud Lat. (Latin)</title>
             <respStmt>
-               <resp>Summary description</resp>
-               <persName>Elizabeth Solopova</persName><persName>Matthew Holford</persName>
+               <resp xml:id="DM">Cataloguer</resp>
```

Comment:
Is this correct? Shouldn't the `xml:id` go on the `respStmt` instead?

https://github.com/bodleian/medieval-mss/blob/master/collections/Add_A/MS_Add_A_100.xml#L21-L25